Data/Sys/GameSettings/HC4.ini

```diff
@@ -0,0 +1,14 @@
+# HC4P01, HC4E9Z, HC4P9Z - Crunchyroll Channel
+
+[Core]
+# Values set here will override the main Dolphin settings.
+MMU = True
+
+[OnLoad]
+# Add memory patches to be loaded once on boot here.
+
+[OnFrame]
+# Add memory patches to be applied every frame here.
+
+[ActionReplay]
+# Add action replay cheats here.
```

Comment:
Missing newline at end of file. There's a dozen or so other files with the same issue, please fix all of them.

Reply:
Will do